mobius/src/algorithms/orbit_ifs.rs

```diff
@@ -0,0 +1,88 @@
+use std::collections::VecDeque;
+
+use abstraction::GroupAction;
+
+use crate::{isogonal::Isogonal, transformable::Transformable};
+
+use super::{
+    point_set::{PointSet, Set},
+    IsogonalTile,
+};
+
+pub struct OrbitIFS {
```

Comment:
I don't particularly like how this is very focused on Isogonal transformations. But given that GA stuff hasn't been finished yet, leave this as-is for now.